tasks/task_manager.hh

```diff
@@ -280,7 +284,7 @@ public:
             impl& operator=(impl&&) = delete;
             virtual ~impl() = default;
         protected:
-            static future<std::vector<task_identity>> get_children(module_ptr module, task_id parent_id);
+            static future<std::vector<task_identity>> get_children(module_ptr module, task_id parent_id, gms::gossiper& gossiper);
```

Comment:
If the only use of the gossiper is to check if the node is alive, maybe just pass down a `std::function<bool(host_id)> is_host_alive>` instead, to avoid the dependency creep.